app/views/pages/release-notes/25-NOVEMBER-2019.liquid

```diff
@@ -0,0 +1,32 @@
+---
+converter: markdown
+metadata:
+  title: Service workers route, multiple queries via GraphiQL editor, improvements, fixes
+  description: November 25, 2019
+  contributors: false
+slug: release-notes/25-NOVEMBER-2019
+searchable: true
+---
+
+#### {{ page.metadata.description }}
+
+<h4 class="release-note release-note__new">NEW</h4>
+
+* special route for service workers
+
+You can put a file named `serviceworker.js` (or `sw.js`) in `app/assets` directory and access it via https://www.example.com/serviceworker.js` (or `https://example.com/sw.js`). In case you need more files accessible this way, you can add numbers to the filename, for example `serviceworker-1.js`, `serviceworker-2.js` etc
```

Comment:
```suggestion
You can put a file named `serviceworker.js` (or `sw.js`) in `app/assets` directory and access it via `https://www.example.com/serviceworker.js` (or `https://example.com/sw.js`). In case you need more files accessible this way, you can add numbers to the filename, for example `serviceworker-1.js`, `serviceworker-2.js` etc
```